voucherify.js

```diff
@@ -235,6 +235,44 @@ module.exports = function(options) {
 
                 return handler.promise;
             }
+        },
+
+        product: {
+            create: function (product, callback) {
```

Comment:
@imvanzen please add methods to manage SKUs. `voucherify.product.sku` -> create, get, update, delete.